src/Akeneo/FreeTrial/back/Infrastructure/Symfony/Resources/public/js/onboarding/appcuses-onboarding.ts

```diff
@@ -0,0 +1,40 @@
+import {getAppcuesAgent} from './appcues-agent';
+import {PimOnBoarding} from 'pimui/js/onboarding';
+
+const FeatureFlags = require('pim/feature-flags');
+const UserContext = require('pim/user-context');
+const Mediator = require('oro/mediator');
+
+const AppcuesOnBoarding: PimOnBoarding = {
```

Comment:
```suggestion
const AppcuesOnboarding: PimOnboarding = {
```

no need to camelcase, it's a word ! ;)